ocfweb/docs/docs/services/account/content-removal.md

```diff
@@ -0,0 +1,63 @@
+[[!meta title="Content Removal Policy"]]
+
+
+## Introduction
+
+Most of the content on [www.ocf.berkeley.edu](https://www.ocf.berkeley.edu)
+is created and owned by OCF members, not the OCF itself. This user-generated
```

Comment:
"user-generated content" seems to be the standard way of saying "stuff we didn't make"